core/chains/evm/log/helpers_test.go

```diff
@@ -58,20 +58,10 @@ type broadcasterHelper struct {
 	pipelineHelper cltest.JobPipelineV2TestHelper
 }
 
-func newBroadcasterHelper(t *testing.T, blockHeight int64, timesSubscribe int, overridesFn func(*chainlink.Config, *chainlink.Secrets)) *broadcasterHelper {
-	return broadcasterHelperCfg{}.new(t, blockHeight, timesSubscribe, nil, overridesFn)
-}
-
-type broadcasterHelperCfg struct {
-	highestSeenHead *evmtypes.Head
-	db              *sqlx.DB
-}
```

Comment:
This was awkward and barely used any longer, so I simplified it away. The db field was no cost, but the tradeoff for `highestSeenHead` is to pass some nils instead :shrug:

Reply:
i don't love required args with nil values, but the surface is small and only in tests so maybe ok.

l like XXXcfg.New() (*XXX) even less...

Reply:
Yeah I thought the same but didn't stress since it's not even exported